### PR TITLE
Added GauravDarkslayer.json

### PR DESCRIPTION
**Pull request by Gauravdarkslayer**
## Changes proposed

Added Gauravdarkslayer.json

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3131"></a>

### CODE REVIEW

File: data/Gauravdarkslayer.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "Gaurav Bhojwani",
+    "type": "personal",
+    "bio": "Keep calm and learn everyday",
+    "avatar": "https://avatars.githubusercontent.com/u/18503497",
```
Comment (Aadarsh805) [inline data/Gauravdarkslayer.json:5]: Hey Gaurav, use this link for the avatar instead `https://github.com/Gauravdarkslayer.png`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Gauravdarkslayer.json (hunk 1 above)
Comment (Gauravdarkslayer) [reply data/Gauravdarkslayer.json:5]: Alright !

**Review by Aadarsh805 — APPROVED** [review]
All good 👍

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Pull request merged**